### PR TITLE
[DOCS-7049] Update upgrade path

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Comment by oliveralfresco2018** [comment]
I've done that @isandhu 
Cheers

**Review by Amy-Hyland — DISMISSED** [review]
Builds and LGTM

File: process-services/latest/upgrade/index.md [hunk 1]
```diff
@@ -85,8 +99,6 @@ Activiti 5 had only 1 job table and this meant that a fairly complex query had t
 
 From Activiti 6, the jobs have been split up in a job `ACT_RU_JOB`, timer `ACT_RU_TIMER_JOB`, suspended `ACT_RU_SUSPENDED_JOB`, and `ACT_RU_DEADLETTER_JOB` dead letter table.
 
-> **Important** Before you upgrade from 1.x to 2.x you must set the `activiti.engine5.enabled` property to `true` in the `activiti-app.properties` file.
-
 Once you have upgraded from 1.x to 2.x you must set the `activiti.engine5.enabled` property to `true` in the `activiti-app.properties` file.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-services/latest/upgrade/index.md]: Done

**Review by isandhu — APPROVED** [review]
Looks good to me.

**Pull request merged**